src/resources/i18n/translations/en.json

```diff
@@ -17,16 +17,16 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount may be higher due to network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add custom RSS feed",
-  "Add custom RSS feeds": "Add custom RSS feeds",
+  "Add current item next in queue when playing a new item": "Start playback immediately before continuing",
```

Comment:
```suggestion
  "Add current item next in queue when playing a new item": "Playback at top of queue",
```

Reply:
I like this suggestion but I think it needs an addition of a verb there since its a CTA

Reply:
The only requirement I can find is to make it short enough.
https://source.android.com/docs/devices/automotive/hmi/car_ui/appendix_b
It isn't a call to action, but rather a setting(?)

Reply:
How about "Add unfinished episode next in queue"?